policyengine_us/parameters/gov/states/ks/tax/income/agi/subtractions/oasdi/agi_limit.yaml

```
@@ -0,0 +1,13 @@
+description: KS federal AGI limit to be eligible for taxable OASDI subtraction.
+values:
+  2021-01-01: 75_000
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: "KS federal AGI limit for taxable social security AGI subtraction"
```

And elsewhere please
```suggestion
  label: "Kansas federal AGI limit for taxable social security AGI subtraction"
```

---

Done